boot/zephyr/main.c

```diff
@@ -60,11 +60,12 @@ void main(void)
 	}
 
 	printk("Bootloader chain: 0x%x\n", rsp.br_image_addr);
-	vt = (struct vector_table *)(rsp.br_image_addr +
-				     rsp.br_hdr->ih_hdr_size);
+	vt = (struct vector_table *)(rsp.br_image_addr + TEXT_SECTION_OFFSET);
```

Comment:
This really needs to be the value read out of the header, which will then work with however it got built. There isn't really a reason to code this value into the bootloader.